hawkbit-repository/hawkbit-repository-core/src/main/java/org/eclipse/hawkbit/repository/StaticQuotaManagement.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2015 Bosch Software Innovations GmbH and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.hawkbit.repository;
+
+import org.eclipse.hawkbit.security.HawkbitSecurityProperties;
+
+public class StaticQuotaManagement implements QuotaManagement {
```

Comment:
1. Add javadoc
2. I would rename the class to `PropertiesQuotaManagement`, because you use properties for the quota